### PR TITLE
[NON-MODULAR] Disable some sanity effects

### PR DESCRIPTION
**Pull request by Floofies**
## About The Pull Request

This PR functionally disables chunks of TG code which implement mechanical buffs/debuffs based on sanity.

I made all of the edits to TG's lines in an auto-merge friendly way by cleanly commenting them out.

## How This Contributes To The Nova Sector Roleplay Experience

This PR removes a lot of the most obtrusive facets of sanity/mood from the game. There is a certain point where it's no longer worth using sanity buffs/debuffs because they are purely mechanics, and as-implemented currently a lot of them are "insufferable" for a roleplay environment. Some of the buffs/debuffs as currently implemented on TG are very unrealistic, such as slower organ healing during sleep, lower crit threshold, and increased tackle damage.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

Having some issues adding files here so here's a link to proof:
https://cdn.discordapp.com/attachments/1172988592658841680/1196596267686580334/dreamseeker_LBb373SogK.mp4

Link to message: https://discord.com/channels/1171566433923239977/1172988592658841680/1196596268672237682

</details>

## Changelog

:cl: A.C.M.O.
balance: Removed sanity effects affecting Settler quirk creature-riding movement.
balance: Removed sanity effects which caused vulnerability to hypnosis.
balance: Removed sanity effects affecting tackles.
balance: Removed sanity effects which were reducing hunger rate.
balance: Removed sanity effects affecting sleep quality, and its effects on organ healing via sleep.
balance: Removed sanity effects affecting disease recovery probability.
balance: Removed sanity effects affecting action speed and movement speed.
balance: Removed sanity effects which were reducing crit thresholds.
/:cl:

### CODE REVIEW

**Comment by OrionTheFox** [comment]
This is a lot of non-modular tweaks for things I've never even seen interacted with period (sleep healing? does anybody use that even on TG?)
I honestly don't think it's worth tweaking, at least not this non-modularly and down here. It's mostly just weird numbers that will never actually see use. The only real one that comes up is movespeed which is understandable

File: code/datums/components/riding/riding_mob.dm [hunk 1]
```diff
@@ -115,6 +115,10 @@
 	var/modified_move_cooldown = vehicle_move_cooldown
 	var/modified_move_delay = vehicle_move_delay
 	if(ishuman(user) && HAS_TRAIT(user, TRAIT_SETTLER))
+		// NOVA EDIT REMOVAL BEGIN - Remove some sanity effects
```
Comment (Iamgoofball) [inline code/datums/components/riding/riding_mob.dm:118]: Please ensure this code is fixed to correctly ensure settlers move at their normal movespeed when riding animals.

**Comment by Iamgoofball** [comment]
You have effectively disabled the mood system.

Just disable it in config at this point.

**Comment by FieryMorty** [comment]
I don't see what's so hard about drinking some soda, eating something nice and enjoying it, smoking a cigarette occasionally, getting and giving a hug.

Then again, one can take the empathy quirk and see 99% of people walking around at the worst moodlet with max starvation.

I think these mechanics are quite realistic and OK to have, just... people don't seem to be very good with mechanics.

**Comment by HoodMilk** [comment]
If you eat food you will have no issue with managing sanity except for accidents (like getting your arms ripped off or surgery without anesthetics, which rightly should make you feel bad). If you want to have the ability to soft disable the mood system just put the nutrient pump implant selectable in character creation and it will let the people who walk around not eating the entire round not have to deal with the bad mood from it.

File: code/datums/components/riding/riding_mob.dm (hunk 1 above)
Comment (Floofies) [reply code/datums/components/riding/riding_mob.dm:118]: You are incorrect. The original code actually speeds Settlers up when they ride animals and have good sanity. With bad sanity it slows them down. The code doesn't actually seem to do what you said it does.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I feel like those sanity effects are inconsequential enough to not really be worth non-modularly disabling them down here. It's the kind of thing that just adds maintenance work during fixing PRs, for very little gain here. I feel like there's probably a better way to go about this, even if I don't have any suggestions right now.

**Comment by Iamgoofball** [comment]
As stated previously, you can just disable mood in the config if you don't want mood.

**Comment by Floofies** [comment]
@Iamgoofball  I'm not config disabling mood in its entirety, hence this PR.

@GoldenAlpharex Do you think it would be a good idea to PR at TG for a new config option then? We want to keep mood/sanity enabled without allowing the most obtuse mechanical effects.

**Comment by Iamgoofball** [comment]
@Floofies You are disabling all impact on the game that Mood has. Ergo, you are disabling mood. Just turn the config off.

**Comment by Floofies** [comment]
@Iamgoofball I'm sorry but that is a discussion between yourself and the maintainers.

**Comment by Floofies** [comment]
OK! After some discussion I am closing this PR due to various problems with it, both technical and balance related.

This turned out to be a great proof-of-concept and taught me some things, though. 🥳 

thankyouchip

**Pull request closed without merging**